bundle/language_en.properties

```diff
@@ -667,6 +677,118 @@ angal.hospital.thevisitinghourvaluesmustbeintherange0to24.msg
+angal.inventory.view.btn                                                                               = View
+angal.inventory.view.btn.key                                                                           = V
+angal.inventory.view.title                                                                             = View Inventory
+angal.inventory.viewinventory.title                                                                    = View Inventory
+angal.inventory.wardinventorybrowser.title                                                             = Ward Inventory Browser
+angal.inventory.youcannotconfirmtheinventorylotrealqtyzero.msg                                         = You cannot confirm the inventory with a new lot having a real quantity of 0.
+angal.inventory.youcannotconfirmthisinventory.msg                                                      = You cannot confirm the inventory with a medicine without a lot.
+angal.inventory.youcannotvalidatetheinventorylotrealqtyzero.msg                                        = You cannot validate the inventory with a new lot having a real quantity of 0.
+angal.inventory.youcannotvalidatethisinventory.msg                                                     = You cannot validate the inventory with a medicine without a lot.
+angal.inventory.youhavealreadyaddedallproduct.msg                                                      = You have already added all the products.
+angal.inventoryrow.doyoureallywanttocleanthistable.msg                                                 = Do you really want to clean this table?
+angal.inventoryrow.doyoureallywanttodeletethisinventoryrow.msg                                         = Do you really want to delete this/those inventory(ies) row(s)?
+angal.inventoryrow.doyoureallywanttovalidatethisinventory.msg                                          = Do you really want to validate this inventory?
+angal.inventoryrow.doyouwanttoaddallnotyetlistedproducts.msg                                           = Do you want to add all not yet listed products?
+angal.inventoryrow.doyouwanttosavethechanges.msg                                                       = Do you want to save the changes?
+angal.inventoryrow.invalidquantity.msg                                                                 = Invalid quantity.
+angal.inventoryrow.lotinformation.title                                                                = Lot Information
+angal.inventoryrow.medicaltype.txt                                                                     = Medical Type
+angal.inventoryrow.pleaseselectatleastoneinventoryrow.msg                                              = Please select at least one inventory row.
+angal.inventoryrow.pleaseselectatleastoneinventoryrow.msg                                              = Please select at least one inventory row.
```

Comment:
Please delete one of these two rows (787 and 788); the key and the text is the same.